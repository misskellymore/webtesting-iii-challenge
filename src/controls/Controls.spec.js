// Test away!

import React from 'react';
import Controls from './Controls.js';

import renderer from 'react-test-renderer';
import { render, fireEvent } from 'react-testing-library';


// Controls snapshot
describe('<Controls />', () => {
    it('should match snapshot', () => {
        const tree = renderer.create(<Controls />).toJSON;

        expect(tree).toMatchSnapshot();
    })

    
})