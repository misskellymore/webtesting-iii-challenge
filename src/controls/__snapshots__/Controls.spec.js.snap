// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Controls /> should match snapshot 1`] = `[Function]`;
